src/components/cylc/table/Table.vue

```diff
@@ -109,6 +109,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             :sort-desc.sync="sortDesc"
             item-key="id"
             show-expand
+            dense
+            :footer-props="{
+              itemsPerPageOptions: [10, 20, 50, 100, 200, -1],
+              showFirstLastPage: true
+            }"
```

Comment:
Looks like you can set the default like so
```suggestion
            }"
            :options="{ itemsPerPage: 50 }"
```